.github/workflows/release.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Publish to Registry
         uses: elgohr/Publish-Docker-Github-Action@47433670daec88d900599790a6fadbf5291a835b
         with:
-          name: lostpixel/lost-pixel-action
+          name: lostpixel/lost-pixel
```

Comment:
Not sure if docker hub supports dashes in the name (need to check)

Reply:
It was with dashes before as well, no?

Reply:
nope, I think the dashes are not allowed in usernames. nor sure.